pyproject.toml

```diff
@@ -51,6 +52,8 @@ tests = [
     "pandas==1.3.5; python_version == '3.7'",
     "pandas==2.0.3; python_version == '3.8'",
     "pandas==2.2.1; python_version > '3.9'",
+    "pyedb==0.5.2; python_version > '3.7'",
+    "pyedb==0.4.2; python_version == '3.7'",
```

Comment:
For the optional dependencies should we allow patch releases too to have flexibility?